src/gei/Commands/MigrateRepoCommand.cs

```diff
@@ -132,6 +135,11 @@ public MigrateRepoCommand(OctoLogger log, ISourceGithubApiFactory sourceGithubAp
             {
                 IsRequired = false
             };
+            var lfsMappingFile = new Option<string>("--lfs-mapping-file")
+            {
+                IsRequired = false,
+                Description = "Rewrite PR SHAs using lfs mapping file during migration."
```

Comment:
```suggestion
                Description = "Rewrite PR SHAs using LFS mapping file during migration (only supported when migrating from GHES)."
```

Reply:
This works when migrating from GHEC also.

Reply:
But there is no archive file when migrating from GHEC? how does that work?

Reply:
Or at least there's no archive file that the CLI has access to AFAIK

Reply:
It works the same as when migrating from GHES. The [metadataArchiveContent is downloaded from GHES or GHEC](https://github.com/robandpdx/gh-gei/blob/robandpdx/src/gei/Commands/MigrateRepoCommand.cs#L322) as a byte[], then I [extract the tar.gz contents to a local directory](https://github.com/robandpdx/gh-gei/blob/robandpdx/src/gei/Commands/MigrateRepoCommand.cs#L483-L486). I then [modify each file extracted as needed](https://github.com/robandpdx/gh-gei/blob/robandpdx/src/gei/Commands/MigrateRepoCommand.cs#L489-L500). Lastly, I create a new [.tar.gz file from the modified files, read that new .tar.gz file into a byte[]](https://github.com/robandpdx/gh-gei/blob/robandpdx/src/gei/Commands/MigrateRepoCommand.cs#L504-L521) and return it to [metadataArchiveContent](https://github.com/robandpdx/gh-gei/blob/robandpdx/src/gei/Commands/MigrateRepoCommand.cs#L327)

Reply:
I plan to do something similar with the [gitArchiveContent](https://github.com/robandpdx/gh-gei/blob/robandpdx/src/gei/Commands/MigrateRepoCommand.cs#L316) to implement an option to convert the repo to LFS during the migration process.